server_addon/README.md

```diff
@@ -0,0 +1,21 @@
+# OpenPype addon for AYON server
+Convert openpype into AYON addon which can be installed on AYON server. The versioning of the addon is following versioning of OpenPype.
+
+## Intro
+As you might know, OpenPype is becoming AYON, which is moving from MongoDB to dedicated server with database. For some time, we'll keep OpenPype compatible with both MongoDB and AYON. But over time we'll start to change codebase to match AYON data and separate individual parts to addons.
+
+At this moment have OpenPype codebase AYON mode, which means that OpenPype is using AYON server instead of MongoDB by under-laying conversion utils. At first, we added AYON executable next to OpenPype executables to start in AYON mode. That works, but would be hard to update new versions of code (would require new full installed build). We've decided to rather create new repository where is only base desktop application logic, for now we call it AYON launcher which will replace executables created by OpenPype build, and convert openpype code to server addon which reduce size of updates.
+
+Because the overall implementation of ayon-launcher is not fully finished yet, we'll keep both ways how to start AYON mode for some time. Once the AYON launcher is finished, we'll remove AYON executables from OpenPype codebase completely.
+
+For some time this addon will be requirement as an entry point to use AYON launcher.
```

Comment:
I've refactored it a little bit with AI just for fun, then edited it a little bit, but I think it is in the end more refined, isn't it?

```suggestion
OpenPype is transitioning to AYON, a dedicated server with its own database, moving away from MongoDB. During this transition period, OpenPype will remain compatible with both MongoDB and AYON. However, we will gradually update the codebase to align with AYON's data structure and separate individual components into addons.

Currently, OpenPype has an AYON mode, which means it utilizes the AYON server instead of MongoDB through conversion utilities. Initially, we added the AYON executable alongside the OpenPype executables to enable AYON mode. While this approach worked, updating to new code versions would require a complete reinstallation. To address this, we have decided to create a new repository specifically for the base desktop application logic, which we currently refer to as the AYON Launcher. This Launcher will replace the executables generated by the OpenPype build and convert the OpenPype code into a server addon, resulting in smaller updates.

Since the implementation of the AYON Launcher is not yet fully completed, we will maintain both methods of starting AYON mode for now. Once the AYON Launcher is finished, we will remove the AYON executables from the OpenPype codebase entirely.

During this transitional period, the AYON Launcher addon will be a requirement as the entry point for using the AYON Launcher.
```